test/turn_test.rb

```diff
@@ -5,4 +5,29 @@
 require './lib/ball'
 
 class TurnTest < Minitest::Test
+
+  def setup
+    @turn = Turn.new(@player, @round)
+    @player = Player.new(@balls, @pins)
+  end
+
+  def test_turn_exist
+    assert_instance_of Turn, @turn
+  end
+
+  def test_
+    assert_equal
+  end
+
+
```

Comment:
you should write tests before writing code

Reply:
For sure